frontend/packages/knative-plugin/integration-tests/features/eventing/eventing-broker-actions.feature

```diff
@@ -0,0 +1,123 @@
+@knative-eventing
+Feature: Perform actions on Broker
+              As a user, I should be able to perform actions on broker
+        
+        
+        Background:
+            Given user has installed OpenShift Serverless Operator
+              And user has created Knative Eventing CR
+              And user has created or selected namespace "aut-eventing-broker-actions"
+              And user is at Topology page
+        
+        
+        @regression @to-do @odc-5030
+        Scenario: Context Menu for Broker: KC-01-TC01
+            Given user has created broker "default" in application groupings "app"
+             When user right clicks on "default" broker
+             Then user will see "Edit Application grouping" option
+              And user will see "Add Trigger" option
+              And user will see "Edit labels" option
+              And user will see "Edit annotations" option
+              And user will see "Edit Broker" option
+              And user will see "Delete Broker" option
+
+        
+        @regression @to-do @odc-5030
+        Scenario: Sidebar for the Broker: KC-01-TC02
+             When user clicks on "default" broker to open the sidebar
+             Then user will see "default" as title of broker
+              And user will see the Resources tab
+              And user will see the Details tab
```

Comment:
I don't think this is required as the below scenarios checking Details and Resources tab individually

Reply:
Removed